src/compute/encryption.rs

```diff
@@ -194,14 +172,11 @@ pub fn encrypt_data(
 
     // Store encrypted AES key in ./0xtask1 [outEncDir]
     write_file(
-        format!("{}/{}", &out_enc_dir, "aes-key.rsa"),
+        format!("{}/aes-key.rsa", &out_enc_dir),
```

Comment:
This seems to work as well
```suggestion
        format!("{out_enc_dir}/aes-key.rsa"),
```

Reply:
Resolved in [3698ecf](https://github.com/iExecBlockchainComputing/tee-worker-post-compute-rust/pull/28/commits/3698ecf74e7ff225326e0fef9bc1e06b5b375d1e)